src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = 40;
+  const fullCost = rentalCost * days;
+  let totalCost = 0;
+
+  if (days >= 1 && days < 3) {
+    totalCost = fullCost;
```

Comment:
Try to simplify you code with return. So move from small to higher rent and return value in each `if` or from function in the end
```suggestion
    return fullCost;
```